server/clip_server/executors/clip_torch.py

```diff
@@ -53,12 +56,13 @@ def __init__(
             # For more details, please see https://pytorch.org/docs/stable/generated/torch.set_num_threads.html
             torch.set_num_threads(max(num_threads, 1))
             torch.set_num_interop_threads(1)
+        self._pool = ThreadPool(processes=num_worker_preprocess)
 
-        self._model, self._preprocess_tensor = clip.load(
-            name, device=self._device, jit=jit
-        )
+        self._model = CLIPModel(name, device=self._device, jit=jit, **kwargs)
+        self._tokenizer = Tokenizer(name)
 
-        self._pool = ThreadPool(processes=num_worker_preprocess)
+        image_size = get_image_size(self._model.visual_model_name)
+        self._image_transform = image_transform(image_size, is_train=False)
```

Comment:
FYI, @shan-mx I changed my mind, I just push a new commit to use original image preprocessor.